doc/man7/property.pod

```diff
@@ -87,7 +87,31 @@ In order to permit a more concise expression of boolean properties, there
 is one short cut: a property name alone (e.g. "default") is
 exactly equivalent to "default=yes" in both definitions and queries.
 
-=head1 Syntax
+=head2 Global and Local
+
+Two levels of property query are supported.
+A context based property query that applies to all fetch operations and a local
+property query.
+Where both the context and local queries include a clause with the same name,
+the local clause is used and the context one ignored.
+For example, a context property query of "fips=yes" and a local property query
+of "fips=no" would result in algorithms that have the "fips" property set t
+"no".
+
+=head2 Override
+
+It is possible for a local property query to override a clause in the context
+property query by preceeding the property name with a '-'.
```

Comment:
I just had another read through, and noticed that this document lacks a complete and unified description of query operators. It's a bit demanding that people should search through the text to *maybe* find out what, for example, `-` means. I would place such a description within the `Queries`, in líst form.

Reply:
Good suggestion, I've added such a list coving the operators and string quotes.